joomla/joomla-cms
# Add package for com_search into 3.10 in preparation for it being decoupled in J4

### Summary of Changes
Adds a core search package for 3.10. This means we don't have to ship it in every update with 4.x for the rest of time. The package is a core extension in 3.10. In 4.0 we will then give it an update server to allow transition to it being a core supported extension

### Testing Instructions
Check new and existing installs of 3.10 can see the search package before and after updating. Ideally on both mysql and postgresql.

### Documentation Changes Required
None
